_posts/2016/2016-03-16-team-orange-mallorca.md

```diff
@@ -0,0 +1,41 @@
+icon: suitcase
+categories: working-at-epages
+authors: ["Alejandra A., Benjamin N. and Heiko A."]
+---
+
+Last week Team Orange, the ePages release and test automation team, was welcomed by the precious views, hearty people and refreshing atmosphere of Mallorca. The purpose of this particular trip was to celebrate an off-site team event liberated from the daily tasks and routines of the business life. The team set off to new shores to develop a master plan on how it could face the winds of change of the advancement of each team member. The ultimate goal was to find the best way for contributing to the promising future of the whole research and development department.
+
+Now we are back and ready to briefly tell you the tale of the fearless developers' journey to a mysterious island. So please enjoy the ambigous report!
+
+{% image blog/blog-mallorca-1.jpg %}{% endimage %}
+
+With much curiosity we first took some time and visited the Drach Caves and the beautiful beach at Porto Cristo and recharged ourselves with inspiration.
+We also visited Sa Calobra and were delighted by the majestic mountains and the bluish tonalities of the Mediterranean Sea.
+
+{% image blog/blog-mallorca-2.jpg %}{% endimage %}
+
+Sharing daily life activities such as having breakfast, shopping, cooking, travelling by car and plane, playing board games and even doing sports together was definitely a delightful way to become closer. Our off-site time was a **BIG** chance to gain trust into the capabilities of our team. We totally adopted this magical present. Unsurprisingly, we soon recognized that living the team spirit can be a lot of fun!
+
+{% image blog/blog-mallorca-3.jpg %}{% endimage %}
+
+There is always time for everything and so we had enough time to work on our technological and business challenges, too. We accomplished two knowledge sharing sessions, one by our pipeline expert in order to understand how it works, another one on language handling and language packs.
+
+{% image blog/blog-mallorca-4.jpg %}{% endimage %}
+
+We also played two rounds of [Coding Dojo](http://codingdojo.org), where we focused on solving the Kata of Conway’s Game of Life. In the first round we tried to apply the constraints “TDD as if you meant it” and “Driver & Navigator“, which strictly seperates the roles in pair programming. The second round focused on shaping distinct, comprehensive and sustainable test cases. Therefore, one ePagee had to implement an unit test. Next, the coding partner should lean into the position of an evil programmer and hence only implement as little as possible to make the test pass. Afterwards he gives back the keyboard to the radically stressed writer of the unit test, who has to enforce the desired implementation with the addition of another test case. In this rotary principle we trained our sense for writing high quality tests. This hands-on approach delivered a lot of amusements as well as practical benefits for our programming skills.
```

Comment:
seperates > separates

...had to implement an unit test. > ... had to implement a unit test.